en/micro-integrator/docs/references/connectors/ldap-connector/ldap-connector-example.md

```diff
@@ -0,0 +1,214 @@
+# LDAP Connector Example
+
+The LDAP connector allows you to connect to any LDAP server through a simple web services interface and perform CRUD 
+(Create, Read, Update, Delete) operations on LDAP entries. This connector uses the JAVA JNDI APIs to connect to a 
+required LDAP server.
+
+Given below is a sample scenario that demonstrates how to perform CRUD operations on LDAP entries using LDAP Connector.
+
+## What you'll build
+
+This example demonstrates on how to use the ldap connector to create and read ldap entries on a student directory. 
+
+![image](../../../assets/img/connectors/ldap_connector/ldap_connector_usecase.png)
+
+This will have 2 API resources, `create`, `search`.
+
+todo : add an image
+
+* `/create` : This will create a new LDAP entry in the LDAP server.
+
+* `/search` : This will performs a search for one or more LDAP entities with the specified search keys.
+
+## Configure the connector in WSO2 Integration Studio
+
+Follow these steps to set up the ESB Solution Project and the Connector Exporter Project. 
```

Comment:
"ESB Solution Project" has been changed to "Integration Project" in the latest tooling. Please change this in all instances in this PR.

Reply:
Fixed in all instances.